OutOfSchool/OutOfSchool.DataAccess/Models/AverageRating.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OutOfSchool.Services.Enums;
+
+namespace OutOfSchool.Services.Models;
+
+public class AverageRating : IKeyedEntity<long>
+{
+    public long Id { get; set; }
+
+    public float Rate { get; set; }
+
+    public int RateQuantity { get; set; }
+
+    [Required]
+    [Range(1, long.MaxValue, ErrorMessage = "The EntityId field should be bigger than 0")]
```

Comment:
should we really have check it as raw number?
maybe investigate something like this [RequiredNonDefault](https://gist.github.com/nickalbrecht/e5972c7018affd6c60e333168f2b5ecc) example.

Reply:
Ok. I've left only the [Required] attribute.